torchci/rockset/metrics/__sql/completed_pr_jobs_aggregate.sql

```diff
@@ -0,0 +1,117 @@
+            INNER JOIN commons.workflow_run r on j.run_id = r.id
+        WHERE
+            1 = 1
+            AND j._event_time > (
+                CURRENT_DATETIME() - DAYS(:from_days_ago)
+            )
+            AND r._event_time > (
+                CURRENT_DATETIME() - DAYS(:from_days_ago)
+            )
+            AND j._event_time < (CURRENT_DATETIME() - DAYS(:to_days_ago))
+            AND r._event_time < (CURRENT_DATETIME() - DAYS(:to_days_ago))
+            AND LENGTH(r.pull_requests) = 1
+            AND r.head_branch NOT IN ('master', 'main', 'nightly')
+            AND r.pull_requests[1].head.repo.name = 'pytorch'
+            AND r.name IN ('pull', 'trunk', 'Lint') -- Ensure we don't pull in random PRs we don't care about
+            AND (
+                r.pull_requests[1].base.ref = 'master'
+                OR r.pull_requests[1].base.ref like 'gh/%/base'
+            )
+        GROUP BY
+            pr_number,
+            url,
+            sha
+    ),
+    -- Now filter the list to just closed PRs
+    merged_pr_shas AS (
+        SELECT  
+            DISTINCT s.pr_number,
+            s.url,
+            s.sha
+        FROM
+            pr_shas s
+            INNER JOIN commons.pull_request pr on s.pr_number = pr.number
+        WHERE
+            pr.closed_at IS NOT NULL 
+            -- Ensure the PR was actaully merged
+            AND 'Merged' IN (
+                SELECT  
+                    name
+                FROM
+                    UNNEST(pr.labels)
+            )
+    ),
+    -- Get all the workflows and partially aggregate the jobs run against each commit (based on the job's conclusion)
+    commit_job_durations AS (
+        SELECT  
+            s.pr_number,
+            r.name AS workflow_name,
+            s.sha,
+            j.conclusion AS conclusion,
+            j.conclusion = 'cancelled' AS was_cancelled, -- For convenience
+            j.run_attempt, -- the attemp # this job was run on
+            r.run_attempt AS total_attempts,
+            r.id AS workflow_run_id,
+            min(j._event_time) AS start_time,
+            max(PARSE_TIMESTAMP_ISO8601(j.completed_at)) AS end_time,
+            DATE_DIFF(
+                'MINUTE',
+                min(j._event_time),
+                max(PARSE_TIMESTAMP_ISO8601(j.completed_at))
+            ) AS duration_mins,
+            CONCAT(
+                'https://github.com/pytorch/pytorch/actions/runs/',
+                r.id
+            ) AS workflow_url, -- for debugging
+            s.url, -- for debugging 
+        FROM
+            commons.workflow_job j
+            INNER JOIN merged_pr_shas s on j.head_sha = s.sha
+            INNER JOIN commons.workflow_run r on j.run_id = r.id
+        WHERE
+            1 = 1
+            AND j._event_time > (
+                CURRENT_DATETIME() - DAYS(:from_days_ago)
+            )
+            AND j._event_time < (CURRENT_DATETIME() - DAYS(:to_days_ago))
+            AND (
```

Comment:
Echo the point above on how `inductor` workflow is handled.  May be we could consider adding `inductor` to the list of retryable workflow https://github.com/pytorch/pytorch/blob/master/.github/pytorch-probot.yml#L16-L21 and read the list here from that instead

Reply:
We could certainly make inductor retryable, but I think it's better to keep inductor out of this metric for the reason stated in the other comment